File: replace_logo.js
const fs = require('fs');
const files = fs.readdirSync('.').filter(f => f.endsWith('.html'));
files.forEach(file => {
    let content = fs.readFileSync(file, 'utf8');
    let newContent = content.replace(/<img[^>]*src="logo_transparent\.png"[^>]*>/g, '<div class="theme-logo"></div>');
    if (content !== newContent) {
        fs.writeFileSync(file, newContent, 'utf8');
        console.log(`Updated ${file}`);
    }
});

File: replace_logo.py
import os
import re

html_files = [f for f in os.listdir('.') if f.endswith('.html')]
pattern = re.compile(r'<img[^>]*src="logo_transparent\.png"[^>]*>')

for file in html_files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    new_content = pattern.sub('<div class="theme-logo"></div>', content)
    
    if content != new_content:
        with open(file, 'w', encoding='utf-8') as f:
            f.write(new_content)
        print(f"Updated {file}")
